docker-compose.yml

```diff
@@ -16,6 +16,7 @@ services:
       - ./resources/deploy/logs:/logs
       - ./resources/deploy/gunicorn.py:/conf/gunicorn.py
       - ./antares-8.2.2-Ubuntu-20.04/bin:/antares_simulator
+    command: ["./scripts/start.sh", "--no-gunicorn"]
```

Comment:
Note: here it will be better to stick with the gunicorn version, because we don't provide the nginx config to do the load balancing (and it's ok, let's keep it simple here).